libsignal-service/src/push_service.rs

```diff
@@ -1058,10 +1058,9 @@ pub trait PushService: MaybeSend {
     // RegistrationSessionMetadataResponse patchVerificationSession(String sessionId, @Nullable String pushToken, @Nullable String mcc, @Nullable String mnc, @Nullable String captchaToken, @Nullable String pushChallengeToken)
     async fn patch_verification_session<'a>(
         &mut self,
+        number: &PhoneNumber,
```

Comment:
This should be optional:

```suggestion
        number: Option<&PhoneNumber>,
```